styles.css

```diff
@@ -0,0 +1,146 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+button:hover{
+    cursor: pointer;
+}
+
+/* Navigation bar */
```

Comment:
Please remove all commented code from your `CSS` file!

Reply:
Hi Oscar. From what I've read before I started the Main Curriculum is that these comments are needed to provide structure and allow ease of reading the code. The comments that make the code messy are code that are commented. Once I find the article, I will send you the link. And thank you for the rest of the feedback.

Reply:
Here is the link: https://speckyboy.com/good-bad-css-practices/
In the "Good practices" section, it mentions making use of a table of content.